deploy/chocolatey/tools/LICENSE.txt

```diff
@@ -0,0 +1,207 @@
+﻿
```

Comment:
is it chocolatey requirement? why we need another license, why can't we push the one from root, what is the difference?

Reply:
Yeah, this is the one packaged in tools hence needed.